doc/api_assets/style.css

```diff
@@ -207,6 +213,12 @@ header h1 {
   padding-top: 1em;
 }
 
+#apicontent .line {
+  width: calc(50% - 1em);
```

Comment:
`calc()` has been supported across the board for a good while: http://caniuse.com/#feat=calc